games-strategy/openxcom/patches/openxcom-1.0.patchset

```diff
@@ -0,0 +1,165 @@
+From 97a047a19dab96d403afba4e4761812158967c52 Mon Sep 17 00:00:00 2001
+From: Mikolaj 'lich' Halber <lich@opmbx.org>
+Date: Sun, 3 Jan 2016 21:31:07 +0000
+Subject: Added Haiku support
+
+
+diff --git a/cmake/modules/FindSDL2.cmake b/cmake/modules/FindSDL2.cmake
+index b407bef..f743a5e 100644
+--- a/cmake/modules/FindSDL2.cmake
++++ b/cmake/modules/FindSDL2.cmake
+@@ -78,7 +78,7 @@
+ FIND_PATH(SDL_INCLUDE_DIR SDL.h
+   HINTS
+   $ENV{SDLDIR}
+-  PATH_SUFFIXES include/SDL include
++  PATH_SUFFIXES include/SDL include headers/x86 headers/x86/SDL
+   PATHS
+   ~/Library/Frameworks
+   /Library/Frameworks
+@@ -90,6 +90,7 @@ FIND_PATH(SDL_INCLUDE_DIR SDL.h
+   /opt/local # DarwinPorts
+   /opt/csw # Blastwave
+   /opt
++  /system/develop/headers/x86/SDL/
```

Comment:
Since you add `headers/x86/SDL` as `PATH_SUFFIXES` I think `/system/develop` and `/home/config/develop` should suffice.